hwcomposer/hwcomposer_backend_v11.cpp

```diff
@@ -72,6 +71,8 @@ HWComposer::HWComposer(unsigned int width, unsigned int height, unsigned int for
     , mlist(mList)
     , num_displays(num_displays)
 {
+    int bufferCount = qBound(2, qgetenv("QPA_HWC_BUFFER_COUNT").toInt(), 8);
```

Comment:
Just a hint of protection against bizarre input, I figured.. Nobody would ever want to use more than 4-5 buffers anyway, so 8 is already a very high number :p